shared/wallets/common/enter-name-popup/index.js

```diff
@@ -2,54 +2,26 @@
 import * as React from 'react'
```

Comment:
this and `enter-key-popup` are dumber now, pretty much inputs with some decoration